### DISCUSSION TITLE
create offer

### DISCUSSION THREAD

**Discussion by AsliddinBuriev** [ProjectOpenSea/opensea-js#1407] [Q&A]
Hi everyone!
I'm new to web3 world. I want to create offer using opensea-js package. 
I'm using 
      ethers: "^5.7.2" && opensea-js: "^6.1.12"
Here is the code: 
      const provider = new ethers.providers.Web3Provider(window.ethereum);
      const sdk = new OpenSeaSDK(
        provider,
        {
          chain: Chain.Klaytn,
          apiKey: 'api key',
        },
        (arg) => console.log(arg),
      );
      const offer = await sdk.createOffer({
        asset: {
          tokenId,
          tokenAddress,
        },
        accountAddress: await this.connector.getAddress(),
        startAmount: price,
        // expirationTime: (Math.round(Date.now() / 1000) + duration).toString(),
      });
      
      But I get 
      "Specified accountAddress is not available through wallet or provider" error. 
      Indeed, when I checked there is no address in provider on creating sdk instance. 
      I dont know how can I get "address" in provider object. I would appreciate any help.

**Comment by ryanio** [comment]
See https://github.com/ProjectOpenSea/opensea-js/blob/main/developerDocs/quick-start.md#wallet
